docs/providers/aws-ec2.md

```diff
@@ -73,7 +73,7 @@ Here is the contents of the aws.env.sh file. Edit it by replacing the placeholde
 # for aws
 export AWS_ACCESS_KEY_ID=<YOUR AWS KEY ID>
 export AWS_SECRET_ACCESS_KEY=<YOUR AWS ACCESS KEY>
-export AWS_KEYPAIR_NAME="aws"
```

Comment:
If we need to change these references, then we need to change https://github.com/terminal-labs/rambo/blob/master/auth/env_scripts/aws.env.sh.dist#L6 as well.

Reply:
This change already exists in the sh setup file.